client/src/partials/reports/modals/openDebtors.modal.html

```diff
@@ -0,0 +1,55 @@
+<form name="ConfigForm" bh-submit="ReportConfigCtrl.generate(ConfigForm)" bh-form-defaults novalidate autocomplete="none">
+
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static">{{ ::ReportConfigCtrl.report.title_key | translate }}</li>
```

Comment:
I'm not sure if this case cannot use the translate directive - if it is possible it would good to bring it in line with standards. 

Potentially this would work:
```html
<li class="static" translate>{{::ReportConfigCtrl.report.title_key}}</li> 
```

This should not block this pull request.